.github/workflows/linkcheck.yml

```diff
@@ -23,9 +23,6 @@ jobs:
         with:
           python-version: "3.12"
           cache: "pip"
-          cache-dependency-path: |
```

Comment:
Note: `cache-dependency-path` is not required. It will search `pyproject.toml` by default. https://github.com/actions/setup-python#caching-packages-dependencies

Reply:
BTW, didn't realize that pyproject is now supported by the Python GitHub action. That's great, and one my previous reasons for holding out over `pyproject.toml` vs. requirements files (though I use `pyproject.toml` internally.